### PR TITLE
Combined PR of AddCalendar testcases: LRQA-71598,  LRQA-71601, LRQA-71604,  LRQA-71606, LRQA-71797, LRQA-71605, LRQA-71595, LRQA-71596, LRQA-71597

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LRQA-71359 (tickets)

There are still other tests.

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/calendar/calendar-web-test/src/testFunctional/tests/AddCalendar.testcase [hunk 1]
```diff
@@ -209,13 +293,49 @@ definition {
...
+	@description = "LRQA-71598 - Verify that the Time Zone selected by default follows the User's preferences."
+	@priority = "3"
+	test DefaultTimeZoneFollowsUserPreferences {
+		Navigator.gotoAccountSettings();
+
+		UserNavigator.gotoPreferences();
+
+		Navigator.gotoNavNested(navNested = "Display Settings");
+
+		PortalSettings.configureTimeZone(timeZone = "(UTC) Coordinated Universal Time");
```
Comment (lucasperj) [inline modules/apps/calendar/calendar-web-test/src/testFunctional/tests/AddCalendar.testcase:305]: Here you can use "(UTC -02:00) Brasilia Summer Time", cause i think (UTC) Coordinated Universal Time) already is the default used. Thanks

File: modules/apps/calendar/calendar-web-test/src/testFunctional/tests/AddCalendar.testcase [hunk 2]
```diff
@@ -209,13 +293,49 @@ definition {
...
+
+		Navigator.gotoPage(pageName = "Calendar Page");
+
+		CalendarNavigator.gotoAddMyCalendars();
+
+		Calendar.viewSelectedTimeZone(timeZone = "(UTC) Coordinated Universal Time");
```
Comment (lucasperj) [inline modules/apps/calendar/calendar-web-test/src/testFunctional/tests/AddCalendar.testcase:311]: Here you may assert the "(UTC -02:00) Brasilia Summer Time", cause i think (UTC) Coordinated Universal Time) already is the default used. Thanks

**Review by lucasperj — CHANGES_REQUESTED** [review]

**Comment by andreefarias** [comment]
ci:test:sf

**Review by lucasperj — APPROVED** [review]

**Comment by lucasperj** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
ci:stop

**Comment by lucasperj** [comment]
ci:stop

**Comment by lucasperj** [comment]
ci:forward

**Pull request closed without merging**